playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -89,12 +89,12 @@ EDXAPP_FOOTER_EXTERNAL_COURSES_LINK:
   url: 'https://online.stanford.edu'
 EDXAPP_FORCED_TPA_PROVIDER_ID: ''
 EDXAPP_FORUM_MONGO_PARAMS:
-  database: "{{ FORUM_MONGO_DATABASE }}"
+  database: "{{ FORUM_MONGO_DATABASE|default('cs_comments_service') }}"
   hosts:
     - host: 'localhost'
-      port: "{{ FORUM_MONGO_PORT }}"
-  password: "{{ FORUM_MONGO_PASSWORD }}"
-  user: "{{ FORUM_MONGO_USER }}"
+      port: "{{ FORUM_MONGO_PORT|default('27017') }}"
+  password: "{{ FORUM_MONGO_PASSWORD|default('password') }}"
+  user: "{{ FORUM_MONGO_USER|default('cs_comments_service') }}"
```

Comment:
These are just the defaults pulled out of `playbooks/roles/forum/defaults/main.yml`.

Note: in all of our multi-instance deploys (stage, lagunita, etc.) these `FORUM_*` vars
are already set via secrets. But here, where we aren't including a secrets file,
we need this to still be defined. This applies to sandbox/devstack.

Reply:
It seems kind of unfortunate that the edxapp role `FORUM_MONGO` vars can't pull from the defaults defined in the forum role, so if forum defaults change, these would need to be changed separately. No way around that I guess?